### PR TITLE
Postgres: schema_evolution = 'auto' by default

### PR DESCRIPTION
**Pull request by simonoswald**
Currently, we do this in a hacky way in `cds deploy` - we should instead do it more centrally in the plugin.

Should fix https://github.com/cap-js/cds-dbs/issues/100

### CODE REVIEW

**Comment by simonoswald** [comment]
I am not 100% sure, but the way I understand it, the plugin will provide the things configured in the package.json as default cds options, right?

File: postgres/package.json [hunk 1]
```diff
@@ -45,15 +46,17 @@
...
           }
         },
         "postgres": {
           "impl": "@cap-js/postgres",
           "dialect": "postgres",
           "vcap": {
             "label": "postgresql-db"
-          }
+          },
+          "schema_evolution": "auto"
```
Comment (johannes-vogel) [inline postgres/package.json:59]: I think it's only required to be set here. The profiles refer to this config via `kind` property.

**Comment by gregorwolf** [comment]
Should be add a test to check this automatically?

**Comment by simonoswald** [comment]
> Should be add a test to check this automatically?

Maybe a check that prints the `cds env` and checks that schema_evolution is set to "auto"? 🤔 Do your tests have access to `cds env`?

**Comment by johannes-vogel** [comment]
> > Should be add a test to check this automatically?
> 
> Maybe a check that prints the `cds env` and checks that schema_evolution is set to "auto"? 🤔 Do your tests have access to `cds env`?

We could also add a similar test that uses schema migration as we do in cds package?

**Comment by simonoswald** [comment]
> > > Should be add a test to check this automatically?
> > 
> > 
> > Maybe a check that prints the `cds env` and checks that schema_evolution is set to "auto"? 🤔 Do your tests have access to `cds env`?
> 
> We could also add a similar test that uses schema migration as we do in cds package?

I think that rebuilding the cds-deploy tests here is a bit overkill

**Pull request merged**